WebInkEnhancement/explainer.md

```diff
@@ -131,7 +131,7 @@ interface InkPresenter {
 
 interface PenStrokeTipPresenter : InkPresenter {
     void setPenStrokeStyle(PenStrokeStyle style);

```

Comment:
I don't recall us discussing this, so I left it for now. However, we just include the radius and color with the setLastRenderedPoint call in the prototype. Should I update the explainer to reflect that, or was that done in the prototype just for simplicity?

Reply:
Discussed in Teams, decided it would be best to keep them all in setLastRenderedPoint as in the prototype.